tests/functional/pages/surveys/checkbox_other/0/checkbox-other-summary.page.js

```diff
@@ -0,0 +1,12 @@
+import QuestionPage from '../../question.page'
```

Comment:
I think it's worth removing as it's a redundant folder, is there ever going to be another version other that 0? Doesn't the number indicate whether it's a live survey or not and since this survey will never be live it will always be zero?